scripts/launcher.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
+
+REM Change code page to UTF-8 for better compatibility.
+@chcp.com 65001 > NUL 
```

Comment:
why not to set the corresponding page according to the file.encoding parameter?

Reply:
Always use UTF-8 for the moment (same behavior with debug console). Consider to improve only if there's feature request for it.